prow/config.yaml

```diff
@@ -51,6 +51,98 @@ presubmits:
     context: pull-kops-e2e-kubernetes-aws
     rerun_command: "/test pull-kops-e2e-kubernetes-aws"
     trigger: "(?m)^/test( all| pull-kops-e2e-kubernetes-aws),?(\\s+|$)"
+  kubernetes-incubator/cri-containerd:
+  - name: pull-cri-containerd-build
+    agent: kubernetes
+    always_run: true
+    context: pull-cri-containerd-build
+    rerun_command: "/test pull-cri-containerd-build"
+    trigger: "(?m)^/test( all| pull-cri-containerd-build),?(\\s+|$)"
+    skip_branches:
+    - v0.1
```

Comment:
or you only want to run it on master?

Reply:
Yeah, only master for now. Let me figure out how to only run on master.

Reply:
```
branches:
- master
```

Reply:
Done.